recipe-server/client/control_new/tests/state/session/test_selectors.js

```diff
@@ -0,0 +1,56 @@
+import { List, Map, is } from 'immutable';
+
+import { getSessionHistory } from 'control_new/state/app/session/selectors';
+import {
+  INITIAL_STATE,
+} from 'control_new/tests/state/session';
+
+const historyItem = (id, category = 'recipe') =>
+  new Map({ url: id, caption: 'caption', category });
+
+let lastId = 0;
+const generateFakeItems = (count, category) => {
```

Comment:
This should be a factory like:
https://github.com/mozilla/normandy/blob/master/recipe-server/client/control_new/tests/state/actions/index.js#L12